site/index.html

```diff
@@ -116,7 +116,13 @@ <h5 class="text-spacing-100"><span class="big">
                   </div>
                 </div>
               </div>
-              <div class="group"><a class="button button-secondary box-with-triangle-right wow fadeScale" href="https://youtu.be/EsCiDSwfAMw" data-lightgallery="item" data-triangle=".button-overlay"><span>Watch promo</span><span class="button-overlay"></span></a></div>
+              <div class="row">
+                <div class="col-md-6 offset-md-3">
+                  <h4 style="font-weight:normal;">No registration needed!</h4>
```

Comment:
Added this info too because I already got many times the same question, even by email!

Reply:
yeap, make a lot of sense, on twitter also many people asked